azure-servicebus/pylintrc

```diff
@@ -0,0 +1,45 @@
+[MASTER]
```

Comment:
Is this the CLI one, or a SB specific one?

Reply:
It is the CLI one with one extra suppression: `useless-object-inheritance`